Sources/MockoloFramework/Models/ParsedEntity.swift

```diff
@@ -99,6 +99,25 @@ final class EntityNodeSubContainer {
     }
 }
 
+public enum CombineSubjectType {
```

Comment:
this and enum below can be merged into one, e.g. 

```
public enum CombineType { 
 case passthroughSubject
 case currentValueSubject 
 case published(propertyWrapper: String, name: String) // or case property(wrapper, name)
}

Reply:
Good call

Reply:
Maybe just ComebineType instead of CombineSubjectType? The latter seems limited to just the first 2 cases.